examples/token-airdrop/src/example.ts

```diff
@@ -0,0 +1,206 @@
+    sequentialInstructionPlan,
+    setTransactionMessageFeePayer,
+    setTransactionMessageLifetimeUsingBlockhash,
+    signTransactionMessageWithSigners,
+} from '@solana/kit';
+import { getCreateAccountInstruction } from '@solana-program/system';
+import { estimateAndUpdateProvisoryComputeUnitLimitFactory, estimateComputeUnitLimitFactory, fillProvisorySetComputeUnitLimitInstruction, getSetComputeUnitPriceInstruction, MAX_COMPUTE_UNIT_LIMIT, updateOrAppendSetComputeUnitLimitInstruction } from '@solana-program/compute-budget';
+import { getMintSize, findAssociatedTokenPda, getCreateAssociatedTokenInstruction, getInitializeMint2Instruction, getMintToCheckedInstruction, TOKEN_PROGRAM_ADDRESS } from '@solana-program/token';
+
+const log = createLogger('Token Airdrop');
+
+/**
+ * SETUP: SOURCE ACCOUNT
+ * The account from which the tokens will be transferred needs to sign the transaction. We need to
+ * create a `TransactionSigner` for it. You can find the account this key relates to in the test
+ * validator fixtures in `/scripts/fixtures/example-transfer-sol-source-account.json`
+ */
+const SOURCE_ACCOUNT_SIGNER = await createKeyPairSignerFromBytes(
+    /**
+     * These are the bytes that we saved at the time this account's key pair was originally
+     * generated. Here, they are inlined into the source code, but you can also imagine them being
+     * loaded from disk or, better yet, read from an environment variable.
+     */
+    new Uint8Array(
+        // prettier-ignore
+        [2, 194, 94, 194, 31, 15, 34, 248, 159, 9, 59, 156, 194, 152, 79, 148, 81, 17, 63, 53, 245, 175, 37, 0, 134, 90, 111, 236, 245, 160, 3, 50, 196, 59, 123, 60, 59, 151, 65, 255, 27, 247, 241, 230, 52, 54, 143, 136, 108, 160, 7, 128, 4, 14, 232, 119, 234, 61, 47, 158, 9, 241, 48, 140],
+    ), // Address: ED1WqT2hWJLSZtj4TtTdoovmpMrr7zpkUdbfxmcJR1Fq
+);
+log.info({ address: SOURCE_ACCOUNT_SIGNER.address }, '[setup] Loaded key pair for source account');
+
+/**
+ * SETUP: RPC CONNECTION
+ * When it comes time to send our transaction to the Solana network for execution, we will do so
+ * through a remote procedure call (RPC) server. This example uses your local test validator which
+ * must be running before you run this script.
+ */
+const rpc = createSolanaRpc('http://127.0.0.1:8899');
+const rpcSubscriptions = createSolanaRpcSubscriptions('ws://127.0.0.1:8900');
+
+// SETUP: destination accounts
+const destinationAddresses = await Promise.all(
+    Array.from({ length: 100 }, async () => {
+        const signer = await generateKeyPairSigner();
+        return signer.address;
+    }),
+);
+log.info('[setup] Generated 100 destination account addresses');
+
+// SETUP: get a blockhash, used for all transactions
+const { value: latestBlockhash } = await rpc.getLatestBlockhash({ commitment: 'confirmed' }).send();
+
+// SETUP: transaction planner
+const transactionPlanner = createTransactionPlanner({
+    createTransactionMessage() {
```

Comment:
Maybe really drive this home with a comment that's like ‘every time the planner needs to create a new transaction message, this function determines the form the base message should take, before any instructions are added by the planner etc etc etc etc’